cibuildwheel/macos.py

```diff
@@ -38,52 +112,21 @@ def build(project_dir, output_dir, test_command, test_requires, test_extras, bef
     get_pip_url = 'https://bootstrap.pypa.io/get-pip.py'
     get_pip_script = '/tmp/get-pip.py'
 
-    pkgs_output = subprocess.check_output(['pkgutil', '--pkgs'], universal_newlines=True)
-    installed_system_packages = pkgs_output.splitlines()
-
-    def call(args, env=None, cwd=None, shell=False):
-        # print the command executing for the logs
-        if shell:
-            print('+ %s' % args)
-        else:
-            print('+ ' + ' '.join(shlex.quote(a) for a in args))
-
-        return subprocess.check_call(args, env=env, cwd=cwd, shell=shell)
-
     # get latest pip once and for all
     download(get_pip_url, get_pip_script)
 
+    symlinks_dir = '/tmp/cibw_bin'
```

Comment:
Hoist this to top-level and call it `SYMLINKS_DIR`? Then we don't need to pass it into install functions